### PR TITLE
Adds Kidan specific emotes.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
Adds two Kidan specific emotes, in line with slimes, synthetics and other races having their own emotes.

Should hopefully add a little more flavour to the currently not so used Kidan race.

https://www.youtube.com/watch?v=BkAYDTJqJvE&feature=youtu.be

The two emotes are *click (*clicks and *clicks-/mob) and *clack (*clacks and *clacks-/mob)

Prints the message "??? clacks their mandibles." and "??? clacks their mandibles at ???" respectively. Clack changes to click for the click sound.

Both sounds are edited by me, originals are free use sounds from http://www.freesound.org/ where most of the games sound effects seem to come from.

TL;DR: Kidan version of Slimes *squish

Edit: Exact sounds used, the correct license to edit and use.  The sounds are used when the emote plays.
http://www.freesound.org/people/newagesoup/sounds/364728/
http://www.freesound.org/people/chaosportal/sounds/137104/

### CODE REVIEW

**Comment by Superhats** [comment]
Ack ack ack!

**Comment by Dumbdumn5** [comment]
CRAAAAAAB PEOPLE CRAAAAB PEOPLE...

**Comment by KasparoVy** [comment]
I wish finding sounds for the other races was this easy. :joy:

**Comment by Fox-McCloud** [comment]
Need to give credit to the exact person you downloaded this from on Freesound and makes a note of it in the comment of where the sound is used and also note that you modified it.

Also needs to make sure the author did include the "you're free to modify this as you please" clause.

**Comment by TravellingMerchant** [comment]
@fox-mccloud ah sure thing! I'll hunt down the exact pages as soon as I get home.

**Comment by TravellingMerchant** [comment]
Added

**Comment by TravellingMerchant** [comment]
Clear to be merged? People seem to like it!

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -163,6 +176,40 @@
...
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if(param)
+				message = "<B>[src]</B> clacks their mandibles at [param]."
+			else
+				message = "<B>[src]</B> clacks their mandibles."
+			playsound(loc, 'sound/effects/Kidanclack.ogg', 50, 0) //Credit to DrMinky (freesound.org) for the sound.
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/emote.dm]: this isn't actually true as these sounds aren't from DrMinky
